### PR TITLE
task 9,10,11 added

### PR DESCRIPTION
**Pull request by nidhi-tagline**

### CODE REVIEW

File: task10.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+# ** **
+
+for i in range(2):
+    for j in range(2-i,0,-1):
+        print('*',end='')
+    for _ in range(i+1):
+        print('  ',end='')
+    for j in range(2-i,0,-1):
+        print('*',end='')
+    print()
```
Comment (himanshu-suthar-tagline) [inline task10.py:16]: Make all the programs generalize. It means, currently the program runs for only one input, but it should run if user enters different value of number.

File: task10.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    
+print()
+# *
+# * *
+# *   *
+# *     *
+# * * * * *   
+for i in range(1,6):
+    for j in range(i):
+        if j == i-1 or j == 0 or i == 5:
```
Comment (himanshu-suthar-tagline) [inline task10.py]: This is a static one, this will not work if I change the n to 10, 11, 12, etc. Please make it generalize.

File: task11.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import math
+
+AP_Series = [2, 5, 8, 11, 15, 17]
+GP_Series = [3, 6, 9, 12, 16, 18]
+
+# For A.P series
+diff = AP_Series[1] - AP_Series[0]   
+for i in range(1,len(AP_Series)):
+    if eval(f'{AP_Series[i]} - {AP_Series[i-1]}') == diff:
```
Comment (himanshu-suthar-tagline) [inline task11.py]: You don't need to use eval method. You can simply performs the subtraction between both of them i.e. 
`AP_Series[i] - AP_Series[i-1]`

File: task11.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+        continue
+    else:
+        AP_Series[i] = AP_Series[i-1] + diff
+print(f'correct A.P : {AP_Series}')
+
+# For G.P series
+ratio = GP_Series[1] / GP_Series[0]   
+for i in range(1,len(GP_Series)):
+    if eval(f'{GP_Series[i]} / {GP_Series[i-1]}') == ratio:
```
Comment (himanshu-suthar-tagline) [inline task11.py]: See the above comment.

**Comment by nidhi-tagline** [comment]
Changes are done.

**Pull request merged**